### DISCUSSION TITLE
How to install without Maven

### DISCUSSION THREAD

**Discussion by tking16** [AppiumTestDistribution/AppiumTestDistribution#1063] [Q&A]
Hi, I'm trying to run parallel tests on a Python framework, is there no npm installation for Appium Test Distribution?

**Comment by MagdelineNg** [comment]
Hi, did you manage to run parallel tests in Python?

**Reply by tking16** [reply]
I did, it's a bit flaky at the moment as I have some timeout issues with WDA being installed on simulators. But the test are running in parallel now. I managed to use Pytest-xdist and Selenium Grid, not AppiumTestDistribution
